Tools/ardupilotwaf/chibios.py

```diff
@@ -88,28 +88,42 @@ def wsl2_prereq_checks(self):
         # 7) doing steps 3 and 4 will most likely take several seconds and in many cases the bootloader has
         #        moved on into the app
         #
-        # Solution: simply call "python.exe" instead of 'python' which magically calls it from the windows
+        # Solution: simply call "python.exe" instead of 'python' which magically calls it from the Windows
         #   system using the same absolute path back into the WSL2's user's directory
-        # Requirements: Windows must have Python3.9.x (NTO 3.10.x) installed and a few packages.
+        # Requirements: Windows must have >= Python3.9.x and module pyserial.
         import subprocess
         try:
+            # raise subprocess.CalledProcessError(1, "foo")
             where_python = subprocess.check_output('where.exe python.exe', shell=True, text=True)
         except subprocess.CalledProcessError:
             # if where.exe can't find the file it returns a non-zero result which throws this exception
             where_python = ""
-        if not where_python or ("\Python\Python" not in where_python and "python.exe" not in where_python):
+        print(flush = True)
```

Comment:
This flush needs to be reworked, it's making CI angry.

Reply:
It can probably not be there. It was an attempt to have the lines from earlier not mix together with these lines. It will all be there. Just a little out of order.